src/open_inwoner/openklant/tests/test_views.py

```diff
@@ -178,6 +180,48 @@ def test_show_detail_for_bsn(self, m):
             },
         )
 
+    def test_show_detail_for_bsn_with_zaak(self, m):
+        data = MockAPIReadData().install_mocks(m, link_objectcontactmomenten=True)
+
+        detail_url = reverse(
+            "cases:contactmoment_detail",
+            kwargs={"kcm_uuid": data.klant_contactmoment["uuid"]},
+        )
+        response = self.app.get(detail_url, user=data.user)
+
+        kcm = response.context["contactmoment"]
+
+        self.assertIsNotNone(response.context["zaak"])
+        self.assertEqual(response.context["zaak"].url, data.zaak["url"])
+        self.assertEqual(response.context["zaak"].identificatie, "Test Zaak")
+        self.assertEqual(
+            kcm,
+            {
+                "registered_date": datetime.fromisoformat(
+                    data.contactmoment["registratiedatum"]
+                ),
+                "channel": data.contactmoment["kanaal"].title(),
+                "text": data.contactmoment["tekst"],
+                "onderwerp": self.contactformsubject.subject,
+                "antwoord": data.contactmoment["antwoord"],
+                "identificatie": data.contactmoment["identificatie"],
+                "type": data.contactmoment["type"],
+                "status": _("Afgehandeld"),
+                "url": detail_url,
+            },
```

Comment:
It'd be a hint more readable and black-friendly if we use short variables so the structures don't become too hairy to read (eg: how it takes `kcm` to not repeat `response.context["contactmoment"]` )